Makefile

```diff
@@ -213,20 +232,28 @@ generate-go: .prefix
 # we first generate the webpack bundle so that bindata knows to atch the
 # output bundle file. then, generate debug bindata source file. finally, we
 # run webpack in watch mode to continuously re-generate the bundle
+.help-short--generate-dev:
+	@echo "Generate and bundle required code in a watch loop"
 generate-dev: .prefix
 	NODE_ENV=development yarn run webpack --progress
 	go run github.com/kevinburke/go-bindata/go-bindata -debug -pkg=bindata -tags full \
 		-o=server/bindata/generated.go \
 		frontend/templates/ assets/... server/mail/templates
 	NODE_ENV=development yarn run webpack --progress --watch
 
+.help-short--generate-mock:
```

Comment:
Since you're actively working on this PR, mind making the tweaks mentioned in [this Slack thread](https://fleetdm.slack.com/archives/C019WG4GH0A/p1734964799330129) on command naming?